features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/tags_api_router.jag

```diff
@@ -65,6 +65,7 @@ routeManager.register('GET', 'publisher', '/publisher/api/tag/{type}/{id}', func
 
     for (var index in tags) {
         tagArray.push({id: counter, name: tags[index]});
+        counter++;
```

Comment:
Are you sure??? For this code I got below output...
```
    for (var index in tags) {
        log.info(index);
        tagArray.push({id: counter, name: tags[index]});
    }
```

[2016-02-04 19:23:03,660]  INFO - tags_api_router:jag 0
[2016-02-04 19:23:03,661]  INFO - tags_api_router:jag 1
[2016-02-04 19:23:03,662]  INFO - tags_api_router:jag 2